### DISCUSSION TITLE
Improvement for documentation

### DISCUSSION THREAD

**Discussion by doginthehat** [remix-run/remix#4705] [Proposals]
Just started exploring Remix, great stuff so far. 

I've found the documentation well written and really useful 😄

On thing I could suggest to improve it... 

there's this very useful section in the documentation https://remix.run/docs/en/v1/api/remix.

It has tons of details about the different components, etc, but there's only a single link to it in the left hand side navigation. 

Every time I need to look in it, it's scroll scroll scroll into that section. 

Would it be possible to place a shortcut to each api item in the left hand side menu?

**Comment by machour** [comment]
I think @ryanflorence is currently enhancing this 👀
